.github/workflows/test_web.yml

```diff
@@ -8,37 +8,33 @@ on:
 
 jobs:
   build:
-    name: Test node ${{ matrix.node_version }} / rust ${{ matrix.rust_version }} / ${{ matrix.os }}
+    name: Test Node.js ${{ matrix.node_version }} / Rust ${{ matrix.rust_version }} / ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     continue-on-error: ${{ matrix.rust_version == 'nightly' }}
     strategy:
       fail-fast: false
       matrix:
         # We most likely don't care about rust versions here, we'll catch those issues in "test rust"
-        node_version: ['10', '12']
+        node_version: ['12', '14']
```

Comment:
Don't know much about Node packages, but is it significant that we're bumping the minimum supported Node version here?

Reply:
Thanks for the feedback.
Node.js 10 is over 2 years old by now, and Node.js 14 is the current LTS. For me it sounds okay.

Reply:
Fair 👍